masurca/4.0.9/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:focal as app
+
+ARG MASURCA_VER="4.0.9"
+ARG MINIMAP2_VER="2.24"
+ARG BWA_VER="0.7.17"
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="masurca"
+LABEL software.version=$MASURCA_VER
+LABEL description="Genome Assembly and Analysis"
+LABEL website="https://github.com/alekseyzimin/masurca"
+LABEL license="https://github.com/alekseyzimin/masurca/blob/master/LICENSE.txt"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+RUN apt-get update && apt-get install -y python \
+  build-essential \
+  curl \
+  wget \
+  gcc \
+  zlib1g-dev \
+  bzip2 \
+  libboost-dev \
+  libbz2-dev \
+  liblzma-dev && \
+  apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# install minimap2
+RUN curl -L https://github.com/lh3/minimap2/releases/download/v${MINIMAP2_VER}/minimap2-${MINIMAP2_VER}_x64-linux.tar.bz2 | tar -jxvf -
+
+# install bwa
+RUN mkdir /bwa && \
+  cd /bwa && \
+  wget https://github.com/lh3/bwa/releases/download/v$BWA_VER/bwa-$BWA_VER.tar.bz2 && \
+  tar -xjf bwa-$BWA_VER.tar.bz2 && \
+  rm bwa-$BWA_VER.tar.bz2 && \
+  cd bwa-$BWA_VER && \
+  make
+
+# install masurca suite
+RUN mkdir /data && \
+  wget https://github.com/alekseyzimin/masurca/releases/download/v$MASURCA_VER/MaSuRCA-$MASURCA_VER.tar.gz && \
+  tar -xvf MaSuRCA-$MASURCA_VER.tar.gz && \
+  rm MaSuRCA-$MASURCA_VER.tar.gz
+
+WORKDIR /MaSuRCA-$MASURCA_VER
+
+# even in 4.0.9 there's the evil cp at the end of .install.sh
+RUN bash /MaSuRCA-$MASURCA_VER/install.sh 2>/dev/null ; exit 0
```

Comment:
oh my lordy, I went down the GitHub Issues rabbit hole with this one 😮 Looks like lots of people have issues installing this software

Reply:
all the more motivation for containerizing this!

Reply:
To be clear, the only reason this install.sh fails is because it tries to copy a directory into a place that already exists. It's literally the last line of the script. Everything else installs fine (from what I gather)

Reply:
👌